cmd/ipi-deprovision/ipi-deprovision.sh

```diff
@@ -1,17 +1,44 @@
 #!/bin/sh
 
+set -o errexit
+set -o nounset
+set -o pipefail
+
+if [ -z "${AWS_SHARED_CREDENTIALS_FILE}" ]; then echo "ENV VAR AWS_SHARED_CREDENTIALS_FILE is not set"; exit 1; fi
+
 ### expirationDate is always with timezone +00:00, eg, "2019-07-23T22:35+0000"
-MY_DATE=$(TZ=":Africa/Abidjan" date '+%Y-%m-%d')
-declare -a regions=("us-east-1" "us-east-2" "us-west-1")
+my_date=$(TZ=":Africa/Abidjan" date '+%Y-%m-%d')
+
+if [[ -n "${IPI_DEPROVISION_DATE_ARG-}" ]]; then
+  ### eg, export IPI_DEPROVISION_DATE_ARG="22 hours ago"
+  echo "take env. var.: IPI_DEPROVISION_DATE_ARG: ${IPI_DEPROVISION_DATE_ARG}"
+  my_date="$(TZ=":Africa/Abidjan" date --date="${IPI_DEPROVISION_DATE_ARG}" '+%Y-%m-%dT%H:%M+0000')"
+fi
+
+echo "my_date: ${my_date}"
+
+### James: CI only runs in us-east-1.
```

Comment:
we don't need to keep this comment around, I don't think

Reply:
Deleted.